.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
ya sorry saw your prev comment after posting mine. And my commits are on a branch fix177. so I guess you mean git rebase -i <branch name>. Then git push --force orgin fix177 right?